README.md

```diff
@@ -2,18 +2,43 @@
 
 RTNS (RTrade Name Service) is a stand-alone IPNS record management service, designed to facilitate secure publishing of IPNS records, leveraging an encrypted keystore known as [kaas](https://github.com/RTradeLtd/kaas). Internally it facilitates scheduled republishing of all published records.
 
-It is essentially a modified and condensed version of [go-ipfs/namesys](https://github.com/ipfs/go-ipfs/tree/master/namesys) with minor optimizations
+It is essentially a modified and condensed version of [go-ipfs/namesys](https://github.com/ipfs/go-ipfs/tree/master/namesys) with minor optimizations.
 
 ## Development
 
-1. Fork and clone the repository
-2. `cd` into the cloned repository
-3. If you cloned the repository into `$GOPATH/src`, run `GO111MODULE=on go mod download` to download the dependencies. If not, run `go mod download`
-4. To run all the tests, use `go test ./...`
+### Using `$GOPATH`
+
+Ref: https://splice.com/blog/contributing-open-source-git-repositories-go/
+
+1. Fork the repository
+2. Clone the repository by running `git clone git@github.com:RTradeLtd/rtns.git $GOPATH/src/github.com/RTradeLtd/rtns`
+3. Run `cd $GOPATH/src/github.com/RTradeLtd/rtns`
+3. Set up remotes.
+
+```bash
+git remote rename origin upstream
+git remote add origin git@github.com:<your-github-username>/rtns.git
+```
+4. Add `export GO111MODULE=on` to `.bashrc` or `.bash_profile` (if you're on a Mac) or `.zshrc` (if you're using [zsh](https://github.com/robbyrussell/oh-my-zsh)). Make sure to reload the rc file of your choice by running `source <rc-file>`
+5. Run `go mod download` to download the dependencies
+6. To run the tests, use `go test ./...`
+
+### Using Go Modules
```

Comment:
Since the `Using $GOPATH` section is also technically using go modules but instead residing in `GOPATH` I would change this to `Outside $GOPATH`

Reply:
Done 🙂